configure.ac

```diff
@@ -64,12 +64,18 @@ AC_ARG_ENABLE([disable-warnings], AS_HELP_STRING([--disable compiler warnings],
 		fi	
 ])
 
+if test "x$CXX" = "xclang++"; then		
```

Comment:
I think it is better to set this in one place - suggest grouping with the test condition on want_cuda as an "or"

Also, lines 8 and 9, suggest
```c++
AC_PROG_CC([icc gcc])
AC_PROG_CXX([icpc g++ gcc])
```
changed too

```c++
AC_PROG_CC([icc gcc clang])
AC_PROG_CXX([icpc g++ clang++])
```
(I don't know why gcc was repeated before.)